nixos/modules/rename.nix

```diff
@@ -37,6 +37,7 @@ in
       The xow package was removed from nixpkgs. Upstream has deprecated
       the project and users are urged to switch to xone.
     '')
+    (mkRemovedOptionModule [ "networking" "liboop" ] "The corresponding package was removed from nixpkgs.")
```

Comment:
This should presumably be `[ "services" "lshd" ]`? Including the reasoning from the package removal would be good too.